views/template/head.ejs

```diff
@@ -2,22 +2,27 @@
 <script async src="https://www.googletagmanager.com/gtag/js?id=UA-127071715-1"></script>
 <script>
     window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
+    function gtag() { dataLayer.push(arguments); }
     gtag('js', new Date());
 
     gtag('config', 'UA-127071715-1');
 </script>
 
 <meta charset="utf-8">
 <title>Coding for Good</title>
-<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico"/>
-<link rel="stylesheet" type="text/css" href="/css/bootstrap.min.css">
-<link rel="stylesheet" type="text/css" href="/css/style.css"/>
-<link rel="stylesheet" type="text/css" href="/css/neutra/stylesheet.css"/>
-<link rel="stylesheet" type="text/css" href="/css/jquery-editable-select.min.css"/>
+<link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+<!-- <link rel="stylesheet" type="text/css" href="/css/bootstrap.min.css"> -->
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
```

Comment:
I thought you have downloaded the bootstrap on local, please then use the local reference